index.html

```diff
@@ -1,11 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=git --help, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.webix.com/edge/webix.css"
+      type="text/css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.materialdesignicons.com/2.7.94/css/materialdesignicons.css"
+      type="text/css"
+      charset="utf-8"
+    />
+
+    <script
+      src="https://cdn.webix.com/edge/webix.js"
+      type="text/javascript"
+    ></script>
+    <link rel="stylesheet" href="./styles/style.css" />
+    <script type="module" src="./script.js"></script>
+    <title>WebixLayout</title>
+  </head>
+  <body>
+    <span
+      >The software is provided by<a href="https://docs.webix.com">webix.com</a>
+      https://docs.webix.com. Allrights reseved (c)</span
```

Comment:
You have two labels like this on the page.
![image](https://user-images.githubusercontent.com/47447469/75444170-00a34580-5974-11ea-94e3-a1fadf539e7a.png)

Reply:
forgot to clean,
fixed it.